frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -808,7 +808,7 @@ const plugin: Plugin<ConsumedExtensions> = [
     type: 'Page/Route',
     properties: {
       exact: true,
-      path: ['/workload-hpa/:ns/:kind/:name'],
+      path: ['/workload-hpa/:ns/:resourceRef/:name', '/workload-hpa/:ns/:resourceRef/:name/edit'],
```

Comment:
hmmm... Should only have one route. If the HPA already exists, it's an Edit scenario since we have no intention to allow a user to add a second HPA ... ?

Reply:
Ah; didn't think of that. Might be a bit odd if the route is invoked from "Add HPA" and ends up on "Edit HPA" but we don't want to allow them to add two+. I'll code that up.